cypress/support/commands.js

```diff
@@ -13,28 +13,13 @@
 // Cypress.Commands.add("login", (email, password) => { ... })
 import '@testing-library/cypress/add-commands';
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
-import existingUser from '../../test-utils/mocks/existingUser';
-import { apiUrl } from '../../common/config/environment';
 
 Cypress.Commands.add('visitAndWaitFor', path => {
   cy.visit(path);
   cy.findByTestId('Desktop Nav').should('exist').and('be.visible');
   cy.url().should('contain', path);
 });
 
-// Use `cy.login()` to quickly login for testing authenticated routes
-Cypress.Commands.add('login', () => {
```

Comment:
I don't love removing this because then we end up slowing down tests and duplicating coverage. Was there problems with maintaining this method? Over time, we could have more tests that require authentication.

Reply:
Yeah I agree, but there is an issue with /profile page.  When you navigate to /profile directly , while already being logged in, React throws hydration errors("Hydration failed because the initial UI does not match what was rendered on the server") and it breaks e2e. Errors are not there if you are redirected from /login or you are navigating with a Link (which I assume most user would do). I've tried to fix the issue but its most likely coming from the wrapper withAuthSync. Maybe refactoring it to functional component or checking for window present, etc would help, but I could not figure it out yet.

Reply:
If it helps you, feel free to drastically alter the logged-in routes. I dont care if it's client rendered, for example.